tests/functional/object_model/server.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, request, command_args, container_name_infix, start_container_
         self.container_log_line = container_log_line
         self.server_log_level = server_log_level
         self.target_device = target_device
+        self.started_by_fixture = request.fixturename
```

Comment:
not used

Reply:
used in stop_by_fixture_name

    @classmethod
    def stop_by_fixture_name(cls, fixture_name):
        instance = list(filter(lambda x: x.started_by_fixture == fixture_name, cls.running_instances))
        if instance:
            logger.debug(f"Stopping server instance spawned by {fixture_name}")
            instance[0].stop()